nservicebus/msmq/distributor/deploying-to-a-cluster.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Deploying distributors to a cluster
+related:
+- nservicebus/msmq/troubleshooting
+reviewed: 2017-05-10
+---
+
+Distributors are a single-point-of-failure if deployed to just a single machine. Make them highly available(HA) on a Windows Failover Cluster to avoid this.
+
+The following procedure is a high-level guide on how to deploy distributors onto a Microsoft Windows Failover cluster. This is necessary to avoid having them as a single-point-of-failure.
+
+## Windows failover cluster setup for distributors
+
+ * Set up a Failover (active/passive) Windows Failover cluster:
+  * [Creating a Cluster in Windows Server 2008](https://blogs.msdn.microsoft.com/clustering/2008/01/18/creating-a-cluster-in-windows-server-2008/)
+  * [Creating a Cluster in Windows Server 2012R2 and Windows Server 2016](https://technet.microsoft.com/en-us/library/dn505754.aspx)
+ * Setup a new cluster group. A cluster Group is a group of resources that have a unique NETBIOS network name and can be addressed externally.
+ * Install clustered Message Queuing (MSMQ) and Distributed Transaction Coordinator(DTC) into the new cluster group
+  * Install the Message Queuing Services on each node
+  * Use the Failover Cluster Manager to add Message Queuing as a clustered role 
```

Comment:
Is there a generic MS docs link we can link to on adding a server role for MSMQ?